esmvalcore/_main.py

```diff
@@ -64,46 +65,51 @@ def get_args():
     parser = argparse.ArgumentParser(
         description=HEADER,
         formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument('recipe', help='Path or name of the yaml recipe file')
     parser.add_argument(
         '-v',
         '--version',
         action='version',
         version=__version__,
         help="return ESMValTool's version number and exit")
-    parser.add_argument(
+    subparsers = parser.add_subparsers(required=True)
+
+    run = subparsers.add_parser("run", help='Run recipe')
+    run.set_defaults(func=main)
+    run.add_argument('recipe', help='Path or name of the yaml recipe file')
+
+    run.add_argument(
         '-c',
         '--config-file',
         default=os.path.join(os.path.dirname(__file__), 'config-user.yml'),
         help='Config file')
-    parser.add_argument(
+    run.add_argument(
         '-s',
         '--synda-download',
         action='store_true',
         help='Download input data using synda. This requires a working '
         'synda installation.')
-    parser.add_argument(
+    run.add_argument(
```

Comment:
Many of the `run` settings are meant for debugging. Would it be an idea to place those in a `debug` command, to indicate clearly that those are mostly meant for diagnostic developers?

Reply:
Maybe